packages/astro/src/assets/utils/index.ts

```diff
@@ -1,4 +1,4 @@
-export { emitESMImage } from './emitAsset.js';
+export { emitESMImage } from './node/emitAsset.js';
```

Comment:
So this is annoying, `emitAsset` (obviously) isn't non-Node safe, but I exported it from the utils without thinking a while ago, and now it means that `astro:assets` can't actually import from `astro/assets/utils` otherwise it breaks runtime.

For now, I moved `emitAsset` to its own folder to reduce damage, and in the future I'll update the Markdoc integration and stuff to import it from the proper place.